wger/gym/templates/gym/member_list.html

```diff
@@ -323,4 +342,4 @@ <h4>{% trans "Emails" %}</h4>
         {% trans "Add member" %}
     </a>
 {% endif %}
-{% endblock %}
+{% endblock %}
```

Comment:
Every file should have a newline at the end.

Reply:
I have fixed this.